enterprise_access/apps/subsidy_access_policy/admin/__init__.py

```diff
@@ -375,4 +376,35 @@ class ForcedPolicyRedemptionAdmin(DjangoQLSearchMixin, SimpleHistoryAdmin):
     readonly_fields = [
         'redeemed_at',
         'errored_at',
+        'transaction_uuid',
+        'traceback',
     ]
+
+    def save_model(self, request, obj, form, change):
+        """
+        If this record has not been successfully redeemed yet,
+        and if ``wait_to_redeem`` is false, then call ``force_redeem()`` on
+        the record.
+        """
+        super().save_model(request, obj, form, change)
+        obj.refresh_from_db()
+
+        if obj.transaction_uuid:
+            message = gettext_lazy("{} has already been redeemed".format(obj))
+            self.message_user(request, message, messages.SUCCESS)
+            return
+
+        if obj.wait_to_redeem:
+            message = gettext_lazy(
+                "{} has wait_to_redeem set to true, redemption will not occur "
```

Comment:
[question/clarification] Unrelated to the specific changes, just curious if you can shed some addtl light on `wait_to_redeem`? What is the expected behavior when `wait_to_redeem` is true?

Reply:
Just that `force_redeem()` won't be called - it's a way to create a record without actually executing the forced redemption.

Reply:
Right. I suppose I'm asking what the use case is to create a record without executing the forced redemption straight away?